scipy/stats/_stats_py.py

```diff
@@ -696,7 +698,7 @@ def tmean(a, limits=None, inclusive=(True, True), axis=None):
     return mean[()] if mean.ndim == 0 else mean
 
 
-@xp_capabilities("trimmed statistics")
+@xp_capabilities(like=tmean)
```

Comment:
I foud that, once you add `static_argnames` to the decorator, the likelyhood of getting exactly identical capabilities between two functions are slim. So I opted for a simpler design that doesn't involve a global state.